XenAdmin/SettingsPanels/GpuEditPage.resx

```diff
@@ -259,7 +256,19 @@
     <value>50</value>
   </data>
   <data name="VideoRAMColumn.Width" type="System.Int32, mscorlib">
-    <value>72</value>
+    <value>90</value>
+  </data>
+  <metadata name="MultiColumn.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+    <value>True</value>
+  </metadata>
+  <data name="MultiColumn.HeaderText" xml:space="preserve">

```

Comment:
I think adding this column doesn't provide additional information to a user. After some vGPUs have been added, all values in this column will show "Yes". If it's "No", no more items can be added. You have no chance to see mixing values in this column.
You may concern how do we tell the user whether a vGPU type supports multiple or not. The plan is to add text in entries of the combo-box of the "adding" dialog. But it hasn't been done yet.